app/src/Components/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { MenuItems } from "./MenuItems"
+import '../Css/Navbar.css'
+import MenuBars from "../Graphics/Icons/MenuBars.png"
+import MenuXmark from "../Graphics/Icons/MenuXmark.png"
+
+class Navbar extends Component {
```

Comment:
Nie używamy klas, to stara technika i nie powinno się już tak pisać. Lepiej użyć funkcji tak jak w innych komponentach.